README.md

```diff
@@ -1,24 +1,165 @@
+<br/>
 
-* Deployment instructions
+# App  Screenshots <a name="screenshots"></a>
+  <img src="app/assets/images/homepage.png" alt="">
 
-* ...
+  <img src="app/assets/images/page1.png" alt="">
+
+  <img src="app/assets/images/page2.png" alt="">
+<details>
+  <summary>Tech stacks</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+
+  </ul>
+</details>
+<br>
+
+## Key Features <a name="key-features"></a>
+
+- **User can create an account..**
+- **Registered users will be able to create new category and operation..**
+- **Users will be able to view operations and category on the application's home page.**
+- **User can login..**
+- **Rspec test for models, controllers and views..**
+
+## 🚀 Project documentation 
+
+
+### Take a look at project's [Live Demo](https://budgetapp-7px9.onrender.com/)
+<h3>NB: Sorry the sign up function is working but you can't receive an email currently to confirm your account on . </h3>
+
+
+</br>
+
+## 📹 [video presentation](https://drive.google.com/file/d/1aRpppoBC4nVBg6iR4VZo6LaX34jCnjJj/view?usp=sharing)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby]("https://www.ruby-lang.org/en/")
+- IDE such as VsCode.
+- Rails needs to be installed on your system.
+
+### SetUp
+
+Clone this Repository to your desired folder:
+
+``` sh
+cd my-folder
+git clone hhttps://github.com/AbdimulhinYussuf3675/Blog-app.git
+```
+### Install
+Install this project with:
+
+``` sh
+cd Blog-app
+bundle install
+```
+
+### Usage
+To run the project, execute the following command:
+``` sh
+rails s
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdimulhin Adan**
+
+- GitHub: [@AbdimulhinYussuf3675](https://github.com/AbdimulhinYussuf3675)
+- Twitter: [@abdimulhin](https://twitter.com/abdimulhin)
+- LinkedIn: [Abdimulhin Adan](https://github.com/AbdimulhinYussuf3675)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- **Set up associations.**
+- **Will add REST API.**
+- **Intergrate with Frontend technology.**
+- **Add desktop design.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like the project, give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
```

Comment:
- [ ] As was requested in the requirements, kindly gives in this section [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).